src/components/presentational/DumbbellChart/DumbbellChart.tsx

```diff
@@ -0,0 +1,96 @@
+import "./DumbbellChart.css";
+import line from './horizontalLine.svg';
```

Comment:
definitely would NOT use an image just for a horizontal line?  Either SVG or a div with a border?